### PR TITLE
[GOV-83] P2G RTP Negative Tests

### PR DESCRIPTION
**Pull request by apurbraj**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -51,7 +51,7 @@ ext {
 
 dependencies {
     implementation 'com.google.code.gson:gson:2.8.9'
-    implementation 'org.mifos:ph-ee-connector-common:1.4.1-SNAPSHOT'
+    implementation 'org.mifos:ph-ee-connector-common:1.7.2-rc.2'
```
Comment (fynmanoj) [inline build.gradle]: Use 1.8.1 snapshot version here. 
If there are unmerged changes in connector common please let me know

**Review by fynmanoj — CHANGES_REQUESTED** [review]
Can we add spotless and checkstyle?

File: build.gradle (hunk 1 above)
Comment (apurbraj) [reply build.gradle]: Updated the version

File: src/main/java/org/mifos/pheeBillPay/zeebe/ZeebeWorkers.java [hunk 2]
```diff
@@ -157,6 +160,18 @@ public void setupWorkers() {
             String body = variables.get(BILL_RTP_REQ).toString();
             String billerId  = variables.get("billerId").toString();
             BillRTPReqDTO billRTPReqDTO = objectMapper.readValue(body, BillRTPReqDTO.class);
+            if(billRTPReqDTO.getPayerFspDetail() != null) {
+                if (billRTPReqDTO.getPayerFspDetail().getPayerFspId().equals("rhino") && billRTPReqDTO.getPayerFspDetail().getFinancialAddress().equals("122333")) {
```
Comment (fynmanoj) [inline src/main/java/org/mifos/pheeBillPay/zeebe/ZeebeWorkers.java]: please donot hardcode these values, read them from application.yml.
Tenants should always be configurable in Paymenthib

File: src/main/java/org/mifos/pheeBillPay/zeebe/ZeebeWorkers.java [hunk 3]
```diff
@@ -157,6 +160,18 @@ public void setupWorkers() {
...
+                    variables.put("payerRtpRequestSuccess", false);
+                    variables.put("errorInformation", "Payer FI was unreachable");
+                }else if(billRTPReqDTO.getPayerFspDetail().getPayerFspId().equals("rhino") && billRTPReqDTO.getPayerFspDetail().getFinancialAddress().equals("1223334444")){
```
Comment (fynmanoj) [inline src/main/java/org/mifos/pheeBillPay/zeebe/ZeebeWorkers.java]: please donot hardcode these values, read them from application.yml.
Tenants should always be configurable in Paymenthib**

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: src/main/java/org/mifos/pheeBillPay/zeebe/ZeebeWorkers.java (hunk 2 above)
Comment (apurbraj) [reply src/main/java/org/mifos/pheeBillPay/zeebe/ZeebeWorkers.java]: Added these values in application.yaml

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**